### DISCUSSION TITLE
Remove the Otter?

### DISCUSSION THREAD

**Discussion by jlteel** [redimp/otterwiki#127] [Q&A] [answered]
Can I remove the otter in the sidebar? It's a little distracting if you're trying use this app professionally.

**Comment by redimp** [comment, marked as answer]
Hey @jlteel, currently you can not disable the Otter being displayed in the sidebar out of the box.

I can understand that you don't want to display the logo or replace it with your own. Will add this feature in the next release (this week).

**Comment by redimp** [comment]
Addendum: You can already hide the otter logo from the sidebar using a [custom.css](https://otterwiki.com/Customization#customcss) with

```css
.sidebar-logo {
    display: none;
}
```

**Reply by jlteel** [reply]
Thank you very much.

**Comment by redimp** [comment]
Released in [2.5.0](https://github.com/redimp/otterwiki/releases/tag/v2.5.0).

**Reply by jlteel** [reply]
I was able to upgrade to 2.5.0 and I see the checkbox, but when I click on the box it appears to be disabled.

**Reply by jlteel** [reply]
I might have gotten in a rush or something. I went back into the settings and the checkbox was checked.

**Reply by redimp** [reply]
Not your fault but mine: There is a typo in the label of the checkbox. Fixed this in e8903cd. Will release a patched image later today.

**Reply by redimp** [reply]
Fix released in [2.5.1](https://github.com/redimp/otterwiki/releases/tag/v2.5.1).
